### ISSUE TITLE
Generated offline HTML Documentation not displaying diagrams using perspectives

### ISSUE THREAD

**Issue by Beerisforwinners** [structurizr/lite#10]
Embedding an image from a certain perspective is working as expected within the documentation in the web application, e.g.:

`![](embed:Containers{perspective=Applicants})`

However, the image is not being embedded within the offline HTML generated using the "Export to offline HTML page" link.

Using latest docker image of structurizr/lite running locally.

**Comment by simonbrowndotje** [comment]
Yes, the interactive embeds (i.e. perspectives and graphs) are more than simple SVG exports, and are therefore excluded from the offline export.

**Comment by Beerisforwinners** [comment]
That's a shame - thinking about perspectives in particular, would it be possible for interactive embeds to export without the additional interactivity as per existing images but with the perspective "in focus"?
